### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Rachael-Graham**
Weekly docs fixes for links, typos, edits, etc.

### CODE REVIEW

File: docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md [hunk 1]
```diff
@@ -5,114 +5,155 @@ description: |
...
+       region: us-east-1
+       secretRef:
+         name: aws-creds
+         namespace: gloo-system
+       roleArn: arn:aws:iam::123456789012:role/lambda-role
+       lambdaFunctions:
+       - lambdaFunctionName: target-name
+         logicalName: target-name
+   ```
+
```
Comment (artberger) [inline docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md:99]: Perhaps optional last step to the verify h2?
```suggestion
3. Optional: [Verify routing](#verify-routing).
```

File: docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md [hunk 2]
```diff
@@ -5,114 +5,155 @@ description: |
...
+       region: us-east-1
+       secretRef:
+         name: aws-creds
+         namespace: gloo-system
+       awsAccountId: "123456789012"
+       lambdaFunctions:
+       - lambdaFunctionName: target-name
+         logicalName: target-name
+   ```
+
```
Comment (artberger) [inline docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md:156]:
```suggestion
3. Optional: [Verify routing](#verify-routing).
```

Comment (artberger) [inline docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md:156]:
```suggestion
3. Optional: [Verify routing](#verify-routing).
```

File: docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md [hunk 3]
```diff
@@ -5,114 +5,155 @@ description: |
...
+       roleArn: arn:aws:iam::123456789012:role/lambda-role
+       lambdaFunctions:
+       - lambdaFunctionName: target-name
+         logicalName: target-name
+   ```
+
+## Resource-based configuration
+
+### AWS configuration
+For this configuration you will need to create a user or role in the authentication account, and a Lambda function in the account that contains the Lambda functions. The Lambda function will have a resource-based policy statement which will allow the user or role in the authentication account to invoke it.
```
Comment (artberger) [inline docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md:103]:
```suggestion
For the AWS configuration, you create a user or role in the authentication account, and a Lambda function in the account that contains the Lambda functions. The Lambda function has a resource-based policy statement which allows the user or role in the authentication account to invoke it.
```

File: docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md [hunk 4]
```diff
@@ -5,114 +5,155 @@ description: |
...
- - To validate that the configuration is correct, you can follow the steps in the [AWS Lambda guide]({{< versioned_link_path fromRoot="/guides/traffic_management/destination_types/aws_lambda#step-3-create-an-upstream-and-virtual-service" >}}) to create a virtual service that routes to the Lambda function via the AWS upstream
\ No newline at end of file
+2. Create the following resources for your Lambda account.
+   1. Choose an existing or create a new Lambda function that you want to route to.
+   2. Define a resource-based policy statement for the function, which allows the user in the authentication account to invoke it.
+      1. In the AWS console, select the Lambda function.
+      2. Click the **Configuration** tab.
+      3. In the sidebar, click the **Permissions** tab.
+      4. In the **Resource-based policy statements** section, click **Add Permissions**.
+         * Select **AWS account** as the entity which will invoke the function.
```
Comment (artberger) [inline docs/content/guides/traffic_management/destination_types/aws_lambda/cross-account.md:124]:
```suggestion
         * Select **AWS account** as the entity which invokes the function.
```

**Review by artberger — APPROVED** [review]

**Review by artberger — APPROVED** [review]

**Review by Nadine2016 — APPROVED** [review]

**Pull request merged**